Tests/CartonTests/StackTraceTests.swift

```diff
@@ -48,111 +49,416 @@ final class StackTraceTests: XCTestCase {
+    let stackTrace = """
+    forEach@[native code]
+
+
+    wasm-stub@[wasm code]
+    <?>.wasm-function[write]@[wasm code]
+    <?>.wasm-function[swift_reportError]@[wasm code]
+    <?>.wasm-function[_swift_stdlib_reportFatalErrorInFile]@[wasm code]
+    <?>.wasm-function[$ss17_assertionFailure__4file4line5flagss5NeverOs12StaticStringV_A2HSus6UInt32VtFySRys5UInt8VGXEfU_yAMXEfU_]@[wasm code]
+    <?>.wasm-function[$ss17_assertionFailure__4file4line5flagss5NeverOs12StaticStringV_A2HSus6UInt32VtFySRys5UInt8VGXEfU_Tm]@[wasm code]
+    <?>.wasm-function[$ss17_assertionFailure__4file4line5flagss5NeverOs12StaticStringV_A2HSus6UInt32VtF]@[wasm code]
+    <?>.wasm-function[$sSayxSicig]@[wasm code]
+    <?>.wasm-function[$s7TestApp5crashyyF]@[wasm code]
+    <?>.wasm-function[$s7TestAppySay13JavaScriptKit7JSValueOGcfU_]@[wasm code]
+    <?>.wasm-function[$s13JavaScriptKit9JSClosureCyACySayAA7JSValueOGccfcAeFcfU_]@[wasm code]
+    <?>.wasm-function[$s13JavaScriptKit9JSClosureCyACySayAA7JSValueOGccfcAeFcfU_TA]@[wasm code]
+    <?>.wasm-function[$sSay13JavaScriptKit7JSValueOGACIeggo_AdCIegnr_TR]@[wasm code]
+    <?>.wasm-function[$sSay13JavaScriptKit7JSValueOGACIeggo_AdCIegnr_TRTA]@[wasm code]
+    <?>.wasm-function[$sSay13JavaScriptKit7JSValueOGACIegnr_AdCIeggo_TR]@[wasm code]
+    <?>.wasm-function[$sSay13JavaScriptKit7JSValueOGACIegnr_AdCIeggo_TRTA]@[wasm code]
+    <?>.wasm-function[$s13JavaScriptKit24_call_host_function_implyys6UInt32V_SPySo10RawJSValueaGs5Int32VADtF]@[wasm code]
+    <?>.wasm-function[_call_host_function_impl]@[wasm code]
+    <?>.wasm-function[_call_host_function]@[wasm code]
+    wasm-stub@[wasm code]
+    swjs_call_host_function@[native code]
+    callHostFunction
+    """.safariStackTrace
+
+    let expected: [StackTraceItem] =
+      [
+//        .init(
+//          symbol: "wasmFs.fs.writeSync",
+//          location: "./entrypoint/dev.js?:35:21",
+//          kind: .javaScript
+//        ),
+//        .init(
+//          symbol: "a/this.wasiImport.fd_write</<",
+//          location: "./node_modules/@wasmer/wasi/lib/index.esm.js?:115:429",
+//          kind: .javaScript
+//        ),
+//        .init(
+//          symbol: "a/this.wasiImport.fd_write<",
+//          location: "./node_modules/@wasmer/wasi/lib/index.esm.js?:115:372",
+//          kind: .javaScript
+//        ),
+//        .init(
+//          symbol: "Z/<",
+//          location: "./node_modules/@wasmer/wasi/lib/index.esm.js?:102:271",
+//          kind: .javaScript
+//        ),
+//        .init(
+//          symbol: "write",
+//          location: "wasm-function[62062]:0x12af331",
+//          kind: .webAssembly
+//        ), .init(
+//          symbol: "swift_reportError",
+//          location: "wasm-function[21654]:0x37c242",
+//          kind: .webAssembly
+//        ), .init(
+//          symbol: "_swift_stdlib_reportFatalErrorInFile",
+//          location: "wasm-function[22950]:0x3e2996",
+//          kind: .webAssembly
+//        ),
```

Comment:
Can these comments be removed, or would you like to keep them for some purpose?

Reply:
I can remove them; I just kept them because they were in the other stack traces but not Safari’s.

Reply:
No problem, a code comment that explains why it was kept there could also work, but doesn't matter anymore 🙂